Source/controls/modifier_hints.cpp

```diff
@@ -184,21 +184,29 @@ void DrawStartModifierMenu(const Surface &out)
 {
 	if (!start_modifier_active)
 		return;
+
+#ifdef __UWP__
+	static const CircleMenuHint DPad(/*isDpad=*/true, /*top=*/_("Quests"), /*right=*/_("Inv"), /*bottom=*/_("Spells"), /*left=*/_("Char"));
+	DrawCircleMenuHint(out, DPad, { PANEL_LEFT + CircleMarginX, PANEL_TOP - CircleTop });
+#else
 	static const CircleMenuHint DPad(/*isDpad=*/true, /*top=*/_("Menu"), /*right=*/_("Inv"), /*bottom=*/_("Map"), /*left=*/_("Char"));
 	static const CircleMenuHint Buttons(/*isDpad=*/false, /*top=*/"", /*right=*/"", /*bottom=*/_("Spells"), /*left=*/_("Quests"));
 	DrawCircleMenuHint(out, DPad, { PANEL_LEFT + CircleMarginX, PANEL_TOP - CircleTop });
 	DrawCircleMenuHint(out, Buttons, { PANEL_LEFT + PANEL_WIDTH - Buttons.Width() - CircleMarginX, PANEL_TOP - CircleTop });
+#endif

```

Comment:
The hints should look at the mapping rather then be hardcoded

Reply:
I have changed the way it is done. now game_controls.cpp or any other platform - like uwp - can create their own lists